example_flight_log.json

```diff
@@ -0,0 +1,97 @@
+{
+  "type": "FeatureCollection",
+  "features": [
+    {
+      "type": "Feature",
+      "geometry": {
+        "type": "LineString",
+        "coordinates": [
+          [
+            -94.79915088090505,
+            44.097927093782694,
+            1
+          ],
+          [
+            -94.79915088090505,
+            44.097927093782694,
+            5
+          ],
+          [
+            -94.79915088090505,
+            44.097927093782694,
+            15
+          ],
+          [
+            -94.79915088090505,
+            44.097927093782694,
+            30
+          ],
+          [
+            -94.79915088090505,
+            44.097927093782694,
+            50
+          ],
+          [
+            -94.79915088090505,
+            44.097927093782694,
+            50
+          ],
+          [
+            -94.79915088090505,
+            44.097927093782694,
+            50
+          ],
+          [
+            -94.79915088090505,
+            44.097927093782694,
+            50
+          ],
+          [
+            -94.79915088090505,
+            44.097927093782694,
+            50
+          ],
+          [
+            -94.79915088090505,
+            44.097927093782694,
+            50
+          ],
+          [
+            -94.79915088090505,
+            44.097927093782694,
+            50
```

Comment:
Nit: Are all of these lat/lngs purposefully identical? I understand that it's just some sample content, but it might be worth plugging in decent data considering this is publicly viewable. No worries if you don't care to follow up on this.

Reply:
I literally used the example flight log that Logan gave me. The only reason I decided to use a flight log is that I chose to demonstrate using the `import_files` GraphQL mutation, and as of right now it only permits importing files of type `FLIGHT_LOG`. I will manually edit this flight log to make the data look a bit more representative.

Reply:
Done: https://github.com/SenteraLLC/file-upload-examples/pull/1/commits/dc93665bf90f66a1e0d906235f69dc71204ded82